Tests/FluentAssertions.Specs/Types/PropertyInfoSelectorSpecs.cs

```diff
@@ -1,14 +1,56 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Reflection;
-
+using FluentAssertions.Types;
 using Internal.Main.Test;
 using Xunit;
 
 namespace FluentAssertions.Specs.Types
 {
     public class PropertyInfoSelectorSpecs
     {
+        [Fact]
+        public void When_property_info_selector_is_created_with_a_null_type_it_should_throw()
+        {
+            // Arrange
+            PropertyInfoSelector propertyInfoSelector;
+
+            // Act
+            Action act = () => propertyInfoSelector = new PropertyInfoSelector((Type)null);
```

Comment:
I'm still uncomfortable directly using the "internal" types in tests, but we have no clear guidance on what we should and should not do.